src/directive.js

```diff
@@ -33,20 +37,39 @@
                         throwNoKeyException();
                     }
 
-                    var callback = function () {
-                        // Notify about the response availability
-                        scope.onSuccess({response: vcRecaptcha.getResponse(scope.widgetId)});
+                    var callback = function (gRecaptchaResponse) {
+                        // Safe $apply
+                        $timeout(function () {
+                            if(ctrl){
+                                ctrl.$setValidity('recaptcha',true);
+                            }
+                            scope.response = gRecaptchaResponse;
+                            // Notify about the response availability
+                            scope.onSuccess({response: gRecaptchaResponse, widgetId: scope.widgetId});
+                        });
+
+                        // captcha session lasts 2 mins after set.
+                        $timeout(function (){
+                            if(ctrl){
+                                ctrl.$setValidity('recaptcha',false);
+                            }
+                            scope.response = "";
+                            // Notify about the response availability
+                            scope.onExpire({widgetId: scope.widgetId});
+                        }, 2 * 60 * 1000);
                     };
 
-                    vcRecaptcha.create(elm[0], scope.key, callback, {
+                    vcRecaptcha.create(elm[0], key, callback, {
 
-                        theme: attrs.theme || null
+                        theme: scope.theme || null
```

Comment:
the difference here is with `attrs`: `theme="{{myTheme}}"` vs with `scope`: `theme="myTheme"` where `myTheme` is a scoped variable set to `"dark"` or `"light"`.  With `attrs` You do have the ability to pass in a string verbatim (`theme="dark"`), but you can still pass a string with `scope` (`theme="'dark'"`)